app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,28 @@
+
+<ul class="list-group d-flex justify-content-between">
+<header><h1> Tasktron 6000 </h1>
+<p><%= link_to "New Task", new_task_path %></p>
+</header>
+    <% @tasks.each do |task| %>
+    <% if task.completed? %>
+    <li class="list-group-item flex-column align-items-start">
+    <strike><h2><%= link_to task.title, task_path(task.id) %></h2></strike>
+    <div class="btn-group float-right" role="group">
+    <button type="button" class="left btn-outline-success"><%= link_to "EDIT", edit_task_path(task[:id]) %></button>
+    <button type="button" class="right btn-outline-danger"><%= link_to "DELETE", task, method: :delete, data: { confirm: "Really, though? This can't be undone." }%></button></div>
+    <p><button type="button" class="btn btn-info"><%= link_to "Uncomplete", complete_task_path(task), method: :patch %></button></p>
+    </p>Completed <%= time_ago_in_words(task.completed_at)%> ago.</p>
```

Comment:
Your `Task` model doesn't have a `completed_at` field.  Instead I would simply make this line:

```erb
<p><%= task.completed ? "Completed": "Incomplete" %></p>
```